powerline/listers/i3wm.py

```diff
@@ -0,0 +1,79 @@
+# vim:fileencoding=utf-8:noet
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+
+import re
+
+from powerline.theme import requires_segment_info
+from powerline.lib.shell import run_cmd
+from powerline.lib.dict import updated
+
+
+conn = None
+XRANDR_OUTPUT_RE = re.compile('^([0-9A-Za-z-]+) connected')
```

Comment:
I think it is better to keep it here for now. Correct location is something like powerline/bindings/wm.py, but adding new file for such a small single function…